README.md

```diff
@@ -149,3 +149,7 @@ An minimal example tracer is provided in the `src/mock_tracer` directory of the
   [ci]: https://travis-ci.org/opentracing/opentracing-javascript
   [cov]: https://coveralls.io/github/opentracing/opentracing-javascript?branch=master
   [npm]: https://www.npmjs.com/package/opentracing
+
+##LICENSE 
+
+Copyright (c) 2016 Resonance Labs, Inc
```

Comment:
?

Reply:
copyright exists in individual source files. This section should as simple as https://github.com/jaegertracing/jaeger-client-java#license.